script.js

```diff
@@ -21,4 +21,187 @@ function closeScreen() {
   }
 }
 
-menuIcon.addEventListener('click', closeScreen);
\ No newline at end of file
+menuIcon.addEventListener('click', closeScreen);
+
+// Card array
+const popData = [
+  {
+    classNO: 'n1',
+    Subtitle: 'Keeping track of \nhundreds of components',
+    Technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    Description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    githubIcon: './images/github-log.svg',
+    liveIcon: './images/live-icon.svg',
+    ProjectLink: 'https://keebot-png.github.io/personal_portfolio/',
+    GitHubLink: 'https://github.com/keebot-png/personal_portfolio',
```

Comment:
- [ ] [OPTIONAL] The logos for live link and source are not displayed properly kindly consider fixing this so the icons are displayed correctly

![Screenshot_20220916_172709](https://user-images.githubusercontent.com/25227022/190686262-036e8df7-93c0-48bf-bd0b-91cdca054232.png)